src/DGEList.py

```diff
@@ -40,29 +39,43 @@ class DGEList(object):
     _field_strip_re = re.compile(r'[\s"]+')
 
     def __init__(
-        self,
-        counts: Optional[np.matrix] = None,
-        samples: Optional[np.array] = None,
-        genes: Optional[np.array] = None,
-        norm_factors: Optional[np.array] = None,
-        group: Optional[np.array] = None,
-        to_remove_zeroes: Optional[bool] = True
+            self,
+            counts: Optional[np.ndarray] = None,
+            samples: Optional[np.array] = None,
+            genes: Optional[np.array] = None,
+            norm_factors: Optional[np.array] = None,
+            group: Optional[np.array] = None,
+            to_remove_zeroes: Optional[bool] = False,
+            filename: Optional[str]=None,
```

Comment:
Why indent twice?

Reply:
`self.filename` is not documented in docstring.

Reply:
"self"  was lining up  to the function name.  I've put it back to a more conventional formatting.